src/openforms/tests/e2e/test_input_validation.py

```diff
@@ -187,6 +187,24 @@ def test_required_field(self):
             expected_ui_error="Het verplichte veld Required bsn is niet ingevuld.",
         )
 
+    def test_elfproef(self):
+        component: Component = {
+            "type": "bsn",
+            "key": "requiredBSN",
+            "label": "Required bsn",
+        }
+
+        self.assertValidationIsAligned(
+            component,
+            ui_input="1234",
+            expected_ui_error="Ongeldig BSN",
+        )
+        self.assertValidationIsAligned(
+            component,
+            ui_input="123456781",
+            expected_ui_error="Ongeldig BSN",
+        )
```

Comment:
there should be only one `assertValidationIsAligned` call per test case, since it already runs two test cases and sets up all the test data under the hood. Probably best to break this up into two separate tests.